src/modules/auth/api/login.ts

```diff
@@ -24,9 +25,11 @@ interface LoginResult {
 }
 
 export const useLogin = () => {
+    const { showNotification } = useNotificationContext();
+
     return useLazyQuery<LoginResult, LoginArgs>(LOGIN, {
         onError(error) {
-            console.error(error.message);
+            showNotification("error", error.message, error.message);
```

Comment:
But why would anyone want to see a bunch of notifications with exactly the same error message? (those are still logged to the console, so won't be fully ignored) The idea with the key was to only update existing one (and move it to the top of the notification stack) if it exists (otherwise it could be fully ignored)